rosidl_typesupport_connext_c/resource/msg__type_support_c.cpp.em

```diff
@@ -344,7 +365,10 @@ if isinstance(type_, AbstractNestedType):
         return false;
       }
 @[    elif isinstance(type_, AbstractWString)]@
-@{      assert False, 'TBD'}@
+      if (!ros_i.data) {
+        rosidl_generator_c__U16String__init(&ros_i);
```

Comment:
We should probably guard against the unlikely case that this call fails (same below). I noticed that the same could be done for regular strings.

Reply:
Since the same is true for strings I would rather not do it right now. Maybe we can delay this to a follow up PR?

Reply:
Sounds good.